cfme/tests/containers/test_ssl_providers.py

```diff
@@ -0,0 +1,152 @@
+from fauxfactory import gen_alphanumeric, gen_integer
+import pytest
+from itertools import product
+from cfme.containers.provider import ContainersProvider
+from os import path
+from utils import testgen
+from utils.path import data_path
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from cfme.web_ui import fill, flash
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda: current_version() < "5.8.0.3")]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
+
+DEFAULT_SEC_PROTOCOLS = 'SSL trusting custom CA', 'SSL without validation', 'SSL'
+HAWKULAR_SEC_PROTOCOLS = 'SSL trusting custom CA', 'SSL without validation', 'SSL'
+
+alphanumeric_name = gen_alphanumeric(10)
+long_alphanumeric_name = gen_alphanumeric(100)
+integer_name = str(gen_integer(0, 100000000))
+PROVIDER_NAMES = alphanumeric_name, long_alphanumeric_name, integer_name
+
+
+@pytest.mark.parametrize('provider_names', PROVIDER_NAMES)
```

Comment:
We agreed (in the cleanup) that for each polarion case we have 1 test item. in this case you running CMP-9836  3 times. Please use loop (over PROVIDER_NAMES) with soft_assert inside the function instead of parametrization. let keep parametriztion for test items only.